rfcs/text/0010-email.md

```diff
@@ -1,68 +1,44 @@
 
 | field | type | description |
 | --- | --- | --- |
-| `email.bcc.addresses` | `wildcard[]` | Addresses of Bcc's |
-| `email.cc.addresses` | `wildcard[]` | Addresses of Cc's |
-| `email.attachments_count` | long | A field outside the flattened structure to control how many attachments are included in the email |
-| `email.attachments` | flattened | A flattened field for anything related to attachments. This allows objects being stored with all information for each file when you have multiple attachments |
-| `email.direction` | keyword | Direction of the message based on the sending and receving domains |
-| `email.sender.address` | wildcard | Senders email address |
-| `email.sender.domain` | wildcard | Domain of the sender |
-| `email.sender.top_level_domain` | keyword | Top level domain of the sender |
-| `email.sender.registered_domain` | wildcard | Registered domain of the sender |
-| `email.sender.subdomain` | keyword | Subdomain of the sender |
-| `email.message_id` | keyword | Internet message ID of the message |
-| `email.reply_to.address` | wildcard | Reply-to address |
-| `email.return_path.address` | wildcard | The return address for the message |
-| `email.size` | long | Total size of the message, in bytes, including attachments |
-| `email.subject` | wildcard | Subject of the message |
-| `email.recipients.addresses` | `keyword[]` | Recipient addresses |
-| `email.domains` | `keyword[]` | domains related to the email |
-
-
-Other ECS fields used together with email usecases:
-| field | description |
-| --- | --- |
-| `event.duration` | The duration related to the email event. Could be the total duration in Quarantine, how long the email took to send from source to destination etc |
-| `event.start` | When the email event started
-| `event.end` | When the email event ended
-| `process.name` | When the event is related to a server or client. Does not take MTA into account which is part of a ongoing discussion |
-| `network.protocol` | Type of email protocol used |
-| `tls.*` | Used for TLS related information for the connection to for example a SMTP server over TLS |
-
-
+| `email.from` | keyword | Stores the `from` email address |
+| `email.timestamp` | keyword (array) | The local date and time the message was written |
+| `email.to` | keyword (array) | The email address(es) of the message recipient(s) |
+| `email.subject` | keyword; `.text` text multi-field | A brief summary of the topic of the message |
+| `email.cc` | keyword (array) | The email address(es) of the carbon copy (CC) recipient(s) |
+| `email.bcc` | keyword (array) | The email address(es) of the blind carbon copy (CC) recipient(s) |
+| `email.content_type` | keyword | Information about how the message is to be displayed. Typically a MIME type |
+| `email.message_id` | wildcard | Unique identifier for the email message that refers to a particular version of a particular message |
+| `email.reply_to` | keyword | Address that replies should be delivered to |
```

Comment:
It should be clarified that this is referring to RFC5322.ReplyTo and is not the same as Return-Path or RFC5321.MailFrom

Reply:
Thanks @ericwbentley 

So, something like this:
| - | - | - |
| - | - | - |
| `email.reply_to` | keyword | Stores the email address provided in the "Reply-To" originator field |

Reply:
I would go as far to explicitly state the RFC in the description to eliminate any confusion. Or at least the name of the field in the headers.

```suggestion
| `email.reply_to` | keyword | Address that replies should be delivered to (RFC5322.ReplyTo) |
```

Reply:
I'm not seeing "ReplyTo" anywhere in the RFC. I do see ["Reply-To" in the originator field section](https://datatracker.ietf.org/doc/html/rfc5322#section-3.6.2). Is that what you're referring to?

So, something like this:
| - | - | - |
| - | - | - |
| `email.reply_to` | keyword | Stores the email address provided in the "Reply-To" originator field (RFC5322) |

Reply:
Trying to improve and clarify this point in https://github.com/elastic/ecs/pull/1219/commits/a856c0a6462d07bb5fe94d2ba7c6fbcd1eb1c87c. Let me know what you think.